required/tools/xr.dtx

```diff
@@ -21,14 +21,49 @@
 %
 % \fi
 % \iffalse
-%% File: xr.dtx Copyright (C) 1993-2019 David Carlisle
+%% File: xr.dtx Copyright (C) 1993-2024 David Carlisle
 %
 %<package>\NeedsTeXFormat{LaTeX2e}
+%<package>\DeclareRelease{v5}{2023-07-04}{xr-2023-07-04.sty}
+%<package>\DeclareCurrentRelease{}{2024-04-10}
 %<package>\ProvidesPackage{xr}
-%<package>         [2023-07-04 v5.06 eXternal References (DPC)]
+%<package>         [2024-04-10 v6.00 eXternal References (DPC)]
 %
 %<*driver>
-\documentclass{ltxdoc}
+\documentclass{l3doc}
+% currently missing in l3doc, copied from array.dtx
```

Comment:
Iif there isn't already we should open an issue to get this added and then removed in various doc files.

Reply:
there is an issue: https://github.com/latex3/latex3/issues/1359

Reply:
tough, even by me .. I  thought I did, but forgot that would have been in latex3 ...